src/app/shared/github/github.service.ts

```diff
@@ -0,0 +1,79 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { EMPTY, Observable, of } from 'rxjs';
+import { delay, expand, map, toArray } from 'rxjs/operators';
```

Comment:
Remove unused import statements
```suggestion
import { expand, map, toArray } from 'rxjs/operators';
```